### PR TITLE
Clean up launch files and edit readme.

### PR DESCRIPTION
**Pull request by helenol**
- Update launch files to actually work with standard tools... ;)
- Enable LIDAR in KITTI dataset instead of stereo (optional)
- Update params in readme
- Add minimal planning example in readme
- Rearrange readme order to make 15% more sense
- Add clear sphere and occupied sphere radii to ros params
- Fix year of publication (lol)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -311,6 +289,140 @@ ICP based refinement can be applied to the poses of the input pointclouds before
...
+
+    <node name="voxblox_node" pkg="voxblox_ros" type="esdf_server" output="screen" args="-alsologtostderr" clear_params="true">
+      <remap from="pointcloud" to="great_sensor/my_pointcloud"/>
+      <remap from="voxblox_node/esdf_map_out" to="esdf_map" />
+      <param name="tsdf_voxel_size" value="$(arg voxel_size)" />
+      <param name="tsdf_voxels_per_side" value="$(arg voxels_per_side)" />
+      <param name="publish_esdf_map" value="true" />
+      <param name="publish_pointclouds" value="true" />
+      <param name="esdf_max_distance_m" value="2.0" />
+      <param name="color_mode" value="color" />
```
Comment (ZacharyTaylor) [inline README.md]: might make sense to make this example file a bit more minimal and remove params that are set to their defaults

File: README.md [hunk 2]
```diff
@@ -311,6 +289,140 @@ ICP based refinement can be applied to the poses of the input pointclouds before
...
+  double distance = 0.0;
+  if (!voxblox_server_.getEsdfMapPtr()->getDistanceAtPosition(position,
+                                                              &distance)) {
+    return 0.0;
+  }
+  return distance;
+}
+```
+
+# Transformations in Voxblox
```
Comment (ZacharyTaylor) [inline README.md]: our readme is getting really long, at some point maybe we should turn it into a wiki and just have short summaries and links in it

File: voxblox_ros/launch/cow_and_lady_dataset.launch [hunk 3]
```diff
@@ -1,8 +1,10 @@
 <launch>
   <arg name="play_bag" default="true" />
-  <arg name="bag_file" default="/path/to/your/file"/>
+  <!-- <arg name="bag_file" default="/path/to/data.bag"/> -->
```
Comment (ZacharyTaylor) [inline voxblox_ros/launch/cow_and_lady_dataset.launch]: why comment the arg?

**Comment by ZacharyTaylor** [comment]
Also could you update the readme table of contents to match section reordering and new sections

File: README.md (hunk 2 above)
Comment (helenol) [reply README.md]: Yeah I agree. Too lazy to do it today though. ;) I think next section we add it's unavoidable.

File: voxblox_ros/launch/cow_and_lady_dataset.launch (hunk 3 above)
Comment (helenol) [reply voxblox_ros/launch/cow_and_lady_dataset.launch]: Because of who I am as a person.

**Pull request merged**